pandas/tests/test_frame.py

```diff
@@ -6320,6 +6319,20 @@ def test_to_csv_from_csv(self):
             xp.columns = lmap(int,xp.columns)
             assert_frame_equal(xp,rs)
 
+        with ensure_clean() as path:
+            # GH 10833 (TimedeltaIndex formatting)
+            dt = pd.Timedelta(seconds=1)
+            df_orig = pd.DataFrame({'data': list(range(10))},
+                                   index=[i*dt for i in range(10)])
+            df_orig.index.rename('timestamp', inplace=True)
```

Comment:
gr8 thanks. Add a separate whatsnew note (with same issue number) for the name lost in conversion